app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1061,4 +1061,9 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         appContext.getPreferenceKey(R.string.pref_key_fxa_signed_in),
         default = false
     )
+
+    var shouldAutofillCCDetails by booleanPreference(
```

Comment:
Let's add a kDoc for this. I think we generally want to add more comments for all settings to make sure it is understood what each setting is easily.

Reply:
Thanks!
Thinking of addind
```
/**
 * Storing the user choice from the "Credit cards" settings for whether save and autofill cards
 * should be enabled or not.
 * If set to `true` when the user focuses on credit card fields in the webpage an Android prompt letting her
 * select the card details to be filled will appear.
 */
```